Modules/_sqlite/prepare_protocol.c

```diff
@@ -30,26 +30,33 @@ pysqlite_prepare_protocol_init(pysqlite_PrepareProtocol *self, PyObject *args,
     return 0;
 }
 
+static int
+pysqlite_prepare_protocol_traverse(PyObject *self, visitproc visit, void *arg)
```

Comment:
If this object cannot hold any Python objects, why is implementing the GC interface?

Reply:
Because we need to visit the type.

Reply:
Quoting Victor:
```
A heap type must respect the following 3 conditions to collaborate with the GC:

    Have the Py_TPFLAGS_HAVE_GC flag;
    Define a traverse function (tp_traverse) which visits the type: Py_VISIT(Py_TYPE(self));
    Instances must be tracked by the GC.
```